typescript/infra/config/environments/mainnet/interchain/abacus/addresses.json

```diff
@@ -0,0 +1,23 @@
+{
+  "bsc": {
```

Comment:
Will users need this info or just us?

Reply:
users would also need this info

Reply:
In that case I think it should be with the consts in the SDK. I wasn't planning on publishing the infra package going forward.

Reply:
ditto